packages/react-core/src/components/ContextSelector/ContextSelector.tsx

```diff
@@ -92,12 +95,15 @@ export class ContextSelector extends React.Component<ContextSelectorProps> {
       menuAppendTo,
       ouiaId,
       ouiaSafe,
+      menuHeader,
       ...props
     } = this.props;
+
     const menuContainer = (
       <div className={css(styles.contextSelectorMenu)}>
         {isOpen && (
           <FocusTrap focusTrapOptions={{ clickOutsideDeactivates: true }}>
+            {menuHeader && <ContextSelectorMenuList isOpen={isOpen}>{menuHeader}</ContextSelectorMenuList>}
```

Comment:
is this blessed by Core? am not seeing anything like that in the examples there @mcoker https://pf4.patternfly.org/documentation/core/components/contextselector